templates/zerver/invite_user.html

```diff
@@ -17,9 +17,9 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
                             placeholder="{{ _('One or more email addresses...') }}">
                         </textarea>
                     </div>
-                    <label class="control-label" for="custom_body">{{ _('Custom invitation message (if you want to add one)') }}</label>
+                    <label class="control-label" for="custom_invite_body">{{ _('Custom invitation message (if you want to add one)') }}</label>
                     <div class="controls">
-                        <textarea rows="2" class="custom_invite_body"
+                        <textarea rows="2" id="custom_invite_body"
                             name="custom_body"
```

Comment:
This block (plus the corresponding CSS change) should be a separate commit from the rest.